src/Core/Grid/Query/OrderQueryBuilder.php

```diff
@@ -115,11 +116,13 @@ public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria)
     {
         $qb = $this->getBaseQueryBuilder($searchCriteria->getFilters());
         if (isset($searchCriteria->getFilters()['new'])) {
-            $qb = $this->addNewCustomerField($qb->addSelect('o.id_order as o_id_order'));
+            $this->addCustomerField($qb);
```

Comment:
May be could you put this call outside the `if` because it is called in two cases (if/else).